tests/SimBriefTest.php

```diff
@@ -97,10 +97,10 @@ public function testApiCalls()
         $this->assertNotNull($flight['simbrief']);
         $this->assertEquals($briefing->id, $flight['simbrief']['id']);
 
-        $url = str_replace('http://', 'https://', $flight['simbrief']['url']);
-        $this->assertEquals(
-            'https://localhost/api/flights/'.$briefing->flight_id.'/briefing',
```

Comment:
What's the reason for this change?

Reply:
When i make the test in a local php session the server are 127.0.0.1:XXXX and not in localhost, i think that if we verify the url path and not the host the test evaluate the same